de.metas.adempiere.adempiere/base/src/main/java/de/metas/process/ProcessInfo.java

```diff
@@ -1356,6 +1373,35 @@ private OutputType getJRDesiredOutputType()
 			return jrDesiredOutputType;
 		}
 
+		public Optional<String> getType()
```

Comment:
make it private

Reply:
why Optional<String>? when the type is null?

Reply:
is never null actually